### DISCUSSION TITLE
How to not effect snackbars/dialogs when using `Get.back`?

### DISCUSSION THREAD

**Discussion by ShookLyngs** [jonataslaw/getx#1679] [Q&A]
Example:
```
// open a GetBar.
Get.showSnackbar(GetBar(title: 'Login Successful'));
// meant to go back to last page, but instead, it made the GetBar close immediately.
Get.back(); 
```

In fact I have one more problem. 
when I logged in the app, I would like to go back to HomePage.
And in the HomePage, it has a couple of tabs, I want to go back to the first tab in the HomePage.

So how can I go back to HomePage without closing all the snackbars, 
and reset my HomePage current tab index to 0, at the same time?

**Comment by mario020mg** [comment]
I think that Get.back() is actually doing it's work properly. 
Can you try to change the order? I mean:

Instead of this:
Get.showSnackbar(GetBar(title: 'Login Successful'));
Get.back(); 

Do this:
Get.back(); 
Get.showSnackbar(GetBar(title: 'Login Successful'));
